File: Java8037/src/com/ssuhanov/Randomizer.java
package com.ssuhanov;

import java.util.Random;

/**
 * Created by ssuhanov on 4/22/17.
 */
public class Randomizer {

    private static Random random = new Random();

    int firstNumber;
    int secondNumber;
    int thirdNumber;
    int fourthNumber;

    public Randomizer() {
        firstNumber = random.nextInt(9) + 1;
        secondNumber = random.nextInt(10);
        thirdNumber = random.nextInt(10);
        fourthNumber = random.nextInt(10);
    }

}

File: CSharp8037/CSharp8037/Randomizer.cs
using System;

namespace CSharp8037
{
	public class Randomizer
	{
		static Random random = new Random();

		public int firstNumber;
		public int secondNumber;
		public int thirdNumber;
		public int fourthNumber;

		public Randomizer()
		{
			firstNumber = random.Next(10);
			secondNumber = random.Next(10);
			thirdNumber = random.Next(10);
			fourthNumber = random.Next(10);
		}
	}
}
